pkg/controller/compliance/compliance_controller.go

```diff
@@ -442,6 +443,13 @@ func (r *ReconcileCompliance) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	if keyValidatorConfig != nil {
+		if _, err := url.Parse(keyValidatorConfig.Issuer()); err != nil {
```

Comment:
I see now that this check is not necessary.
The constructor for KeyValidatorConfig will create a well known config, which is obtained through making a request to the URL.
This means that we can remove this check again.